### PR TITLE
Fix missing attrs with `svg-in-mustache`

### PR DESCRIPTION
**Pull request by choumx**
And add a visual regression test.

### CODE REVIEW

File: test/functional/test-sanitizer.js [hunk 1]
```diff
@@ -250,6 +250,26 @@ function runSanitizerTests() {
       expectEqualNodeLists(actual, expected);
     });
 
+    it('should output "layout" attribute', () => {
+      const img = '<amp-img layout="responsive"></amp-img>';
+      expect(sanitizeHtml(img)).to.equal(img);
+    });
+
+    it('should output "media" attribute', () => {
```
Comment (aghassemi) [inline test/functional/test-sanitizer.js]: huh, curious how `media` is working without it being whitelisted. There is a standard `media` attribute but only for `style` and `link`. Might be worth listing it explicitly.

Comment (choumx) [reply test/functional/test-sanitizer.js]: Both Caja and DOMPurify whitelist it broadly. DOMPurify doesn't have tag-specific whitelists to begin with, which is what helps keep its size small.

**Comment by choumx** [comment]
Thanks Ali!

**Pull request merged**